DigitalLearningSolutions.Web/Styles/trackingSystem/contractDetails.scss

```diff
@@ -0,0 +1,23 @@
+﻿@import "~nhsuk-frontend/packages/core/all";
+
+.coloured-card-number {

```

Comment:
Can we add a comment here and to the contract details ViewModel that these colour values are interlinked and if someone changes the name here, they should also change it in the other class? :)